flow/designs/asap7/eth_top/config.mk

```diff
@@ -0,0 +1,50 @@
+$(info [INFO-FLOW] Ethernet Design)
+
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NAME            = eth_top
+export DESIGN_NICKNAME        = eth_top
+export DESIGN                 = eth_top
+
+export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
+export SDC_FILE               = $(DESIGN_DIR)/constraint.sdc
+export ABC_AREA               = 1
+
+export CORNER                ?= BC
+
+export LIB_FILES             += $($(CORNER)_LIB_FILES)
+export LIB_DIRS              += $($(CORNER)_LIB_DIRS)
+export DB_FILES              += $($(CORNER)_DB_FILES)
+export DB_DIRS               += $($(CORNER)_DB_DIRS)
+export WRAP_LIBS             += $(WRAP_$(CORNER)_LIBS)
+export WRAP_LEFS             += $(WRAP_$(CORNER)_LEFS)
+export TEMPERATURE            = $($(CORNER)_TEMPERATURE)
+export DFF_LIB_FILE = $($(CORNER)_DFF_LIB_FILE)
+
+export ABC_CLOCK_PERIOD_IN_PS = 400
```

Comment:
Your sdc has -period 1000

Reply:
This is opensource design. We don't have design constraints, just set clock to  meet Positive slack.

Reply:
I don't understand. Then why set this value at all? The flow will extract the value from the SDC. If there isn't a good reason for setting this to 400 then it should be removed.

Reply:
@rovinski is ABC_CLOCK_PERIOD and .sdc clock_period should be same to leverage tool to pick standard cells from library?
Also as per my understand synthesis stage we're not doing any optimization.

Reply:
By default `ABC_CLOCK_PERIOD_IN_PS` is set to the clock from the `.sdc`. Unless there is reasoning to set `ABC_CLOCK_PERIOD_IN_PS` to something different, such as to meet a specific slack target, then it should not be set. It doesn't sound to me like there is a reason.